.github/workflows/ci-rust.yml

```diff
@@ -75,6 +75,11 @@ jobs:
           save-if: ${{ github.ref == 'refs/heads/master' }}
         timeout-minutes: 5
 
+      # Install fuse
+      - name: Install fuse
+        shell: bash
+        run: sudo apt-get -y install fuse3 libfuse3-dev
```

Comment:
Do something like the step `Install postgres` in ci-python

Reply:
Why ?

Reply:
But we don't fail with fuse for the moment no ?

Reply:
Done